sbt/README.md

```diff
@@ -0,0 +1,17 @@
+# SBT
+
+[sbt](1) is a build tool for Scala, Java, and more.
+
+## Maintainers
+
+The Habitat Maintainers: <humans@habitat.sh>
+
+## Usage
+
+This is binary package. To use `sbt` in the dev shell:
+
+```
+hab pkg binlink core/maven mvn
```

Comment:
I think this should read something like:

```
hab pkg install core/sbt --binlink
```

Reply:
Ops... Actually, I have tried running this and turns out I also need to binlink `core/jre8` in order for the `sbt` executable to work. Is there a way to `--binlink` them both in one go?